qiskit/schemas/examples/backend_configuration_openpulse_example.json

```diff
@@ -49,7 +49,7 @@
     "meas_lo_range": [[6.5,7.5],[6.5,7.5],[6.5,7.5]],
     "dt": 1,
     "dtm": 10,
-    "rep_times": [1000,100,50],
+    "rep_times": [1000.0,100.0,50.0],
```

Comment:
Since this is a mocked example of what we receive from the backends, and we haven't updated the backends, this should remain as it was

Reply:
All the jsons should be able to revert to their originals